data/event-schemas/schema/m.room.create.yaml

```diff
@@ -6,7 +6,9 @@ properties:
   content:
     properties:
       creator:
-        description: The `user_id` of the room creator. This is set by the homeserver.
+        description: |-
+          The `user_id` of the room creator. This is set by the homeserver for room versions 1 - 10. Room versions
+          11 no longer includes this property, the event `sender` should be used instead.
```

Comment:
comma splice. And I would say something like "only relevant in room versions 1-10."